docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,167 @@ sidebar_position: 1
 
 ## Compatiblity Note
 
-## Operator
+## Install on Kubernetes
 
-_How can To be provided soon_
+1. Install Operator Lifecycle Manager (OLM), a tool to help manage the Operators running on your cluster.
+
+```
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.1/install.sh | bash -s v0.18.1
+```
+
+2. Install Service Binding Operator
+
+```
+kubectl create -f https://operatorhub.io/install/service-binding-operator.yaml
+```
+
+The operator will be installed in the "operators" namespace and will be usable from all other namespaces in the cluster.
+
+3. Check if operator has been installed successfully:
+
+```
+kubectl get csv -n operators
```

Comment:
Would it be beneficial to show the output of this?

Reply:
Yes. Added.